src/wallet/api.rs

```diff
@@ -639,70 +640,34 @@ impl Wallet {
     }
 
     /// Import watch addresses into core wallet. Does not check if the address was already imported.
-    pub(super) fn import_addresses(
+    pub fn import_descriptors(
```

Comment:
Let's move this API from wallet to RPC module. Eventually, this will be part of the `Blokchain` trait along with sync and other APIs..

Reply:
Yup